RPI2/Ubuntu/clean-for-imaging

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [ $(id -u) -ne 0 ]; then
+   echo >&2 "Must be run as root"
+   exit 1
+fi
+
+set -e
+set -x
+
+. config.env
+
+pushd ../../Common/Ubuntu/
+ ./clean-for-imaging
+popd
+
+rm -f /home/$NORMAL_USER/.ssh/authorized_keys
+
+dd if=/dev/zero of=$HOME/zeroes || true
```

Comment:
dd if=/dev/zero of=$HOME/zeroes status=progress

Reply:
This part is from @peterbarker so I'll let him comment..

Reply:
@rmackay9 this just adds a comforter so you know things are moving along.

Wouldn't hurt.